post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -6,15 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[IMPORTANT]
-====
-The Multiarch Tuning Operator is a Technology Preview feature only. Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
-====
-
 The Multiarch Tuning Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration.
 
-This Operator implements the `clusterpodplacementconfigs` custom resource (CR) to support architecture-aware workload scheduling.
+Scheduling pods on nodes that support the architecture of the pod's images is called architecture-aware workload scheduling. This Operator implements the `clusterpodplacementconfigs` custom resource (CR) to support architecture-aware workload scheduling.
```

Comment:
```suggestion
Architecture-aware workload scheduling allows a cluster to assign pods to nodes that match the architecture of the pod's images. The Kubernetes scheduler does not natively consider the architecture of a pod's images when selecting nodes for placement. This Operator implements the clusterpodplacementconfigs custom resource (CR) to enable architecture-aware scheduling.
```

What about this ^?

Reply:
It looks good! Updated as per the suggestion.